pom.xml

```diff
@@ -1257,6 +1243,17 @@ under the License.
 		<pluginManagement>
 			<plugins>
 
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-release-plugin</artifactId>
```

Comment:
given that it is only defined in the plugin management section i was wondering whether we need this plugin at all.